scripts/start.js

```diff
@@ -27,30 +29,18 @@ fs.readFile( PORT_FILE, ( err, data ) => {
 			// We have not found a port.
 			return;
 		}
+		const protocol = process.env.HTTPS === 'true' ? 'https' : 'http';
 
-		try {
-			fs.writeFileSync( PORT_FILE, port );
-		} catch ( err ) {
-			console.log( chalk.red( `Unable to write port to ${ PORT_FILE }` ) );
-			return;
-		}
+		// Pass in the full hostname of the dev server.
+		overrideConfigCache( `${ protocol }://${ HOST }:${ port }` );
+
+		// Pass the selected port forward so that react-scripts' start will not re-prompt.
+		process.env.PORT = port;
```

Comment:
This should also set `process.env.BROWSER` to avoid launching the useless URL.

Reply:
did so on line below, which didn't dismiss the notice :)